cmd/client/main.go

```diff
@@ -174,7 +174,7 @@ func loop(c Coordinator, client *http.Client) error {
 		level.Error(c.logger).Log("msg", "Error parsing url:", "err", err)
 		return errors.Wrap(err, "error parsing url")
 	}
-	u, err := url.Parse("poll")
+	u, err := url.Parse("poll/" + *myFqdn)
```

Comment:
There should be a flag to not use the FQDN in the poll (yet) in case the server hasn't been updated.

Reply:
would not the `/poll` endpoint of an old server still receive the request from a new client?

the [http.ServerMux](https://golang.org/src/net/http/server.go#L2279) will match against the longest string, which means `/poll` will continue to work and handle even new clients.

Reply:
Thank you for the explanation. I wasn't aware of `http.ServerMux`' longest-prefix matching behaviour.

Reply:
yeah, I had also forgotten about it in the time between creating this PR and now :)